CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## unreleased
 
-Nothing.
+*  Clarify that comments with newlines are allowed before commas in arrays.
```

Comment:
This change isn't just comments with newlines, but comments _and/or_ newlines, right?

Nit-pick, I guess, since this is just the changelog.

Reply:
Oh I didn't check if newlines on their own would be allowed. They should be, since arrays permit them between values. Maybe we need to bring this up in #766.

Reply:
You're right. The change to the ABNF does allow newlines on their own. So both the `CHANGELOG.md` and `toml.md` require changes.

Reply:
I think the changes in toml.md are already sufficient... it reads correctly to me anyways O_o